configuration/cdap-site.xml

```diff
@@ -609,7 +738,7 @@
     <name>metadata.updates.kafka.broker.list</name>
     <value>127.0.0.1:${kafka.bind.port}</value>
     <description>
-      Kafka broker list to which metadata update notifications are published
+      CDAP Kafka service broker list to which metadata update notifications are published
```

Comment:
Actually this can be any Kafka... CDAP's or an external kafka.  So remove the leading ``CDAP``.